### ISSUE TITLE
offline support

### ISSUE THREAD

**Issue by alicewriteswrongs** [matze/iridium#28]
Hello! This looks like a very nice project. I'm wondering, the project description mentions online support but I can't find a way in the application to bypass the Standard Notes server login screen. Am I just something super obvious? Thanks!

**Comment by matze** [comment]
You are right, it's probably not super obvious but if you just use the `Create` instead of the `Sign Up` or `Login` button there will be no registration/connection to any server.

**Comment by alicewriteswrongs** [comment]
ah, neat! if the app is used that way where does it save the notes?

**Comment by matze** [comment]
The program follows the [XDG conventions](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and most likely stores the notes under `~/.local/share/iridium`. Please be aware that the first directory level is the base64 encoded identifier and all the notes are stored encrypted.
